newscout_web/api/v1/urls.py

```diff
@@ -67,6 +68,8 @@
         name='article-create-update'),
     url(r'^article/status/$', ChangeArticleStatusView.as_view(),
         name='article-status'),
-    url(r'daily-digest/$', GetDailyDigestView.as_view(), name='daily-digest')
+    url(r'daily-digest/$', GetDailyDigestView.as_view(), name='daily-digest'),
+    url(r'article/draft-image/$', DraftMediaUploadView.as_view(), name='draft-image'),
+    url(r'article/draft-image/(?P<pk>[-\d]+)/$', DraftMediaUploadView.as_view(), name='draft-image'),
```

Comment:
Consolidate `article/draft-image` and `article/draft-image/(?P<pk>[-\d]+)` into one